common/src/app/common/files/changes.cljc

```diff
@@ -366,9 +366,33 @@
      [:type [:= :del-typography]]
      [:id ::sm/uuid]]]
 
-   [:update-active-token-themes
-    [:map {:title "UpdateActiveTokenThemes"}
-     [:type [:= :update-active-token-themes]]
+   [:set-tokens-lib
+    [:map {:title "SetTokensLib"}
+     [:type [:= :set-tokens-lib]]
+     [:tokens-lib ::sm/any]]]  ;; TODO: we should define a plain object schema for tokens-lib
+
+   [:set-token
+    [:map {:title "SetTokenChange"}
+     [:type [:= :set-token]]
+     [:set-id ::sm/uuid]
+     [:token-id ::sm/uuid]
+     [:token [:maybe ctob/schema:token-attrs]]]]
```

Comment:
My soft suggestion here is use a more generic `:attrs` instead of `:token`. Because we really sent attrs and then create an instance of token after validating that attrs.

Reply:
The same applies to the rest of changes.

Reply:
yep, and make it consistent using datafy in each case